monkey/infection_monkey/post_breach/job_scheduling/linux/job_scheduling.py

```diff
@@ -0,0 +1,11 @@
+TEMP_CRON = "$HOME/monkey-schedule-jobs"
```

Comment:
no need to create a dir for a single file. It's better to do `job_scheduling_linux.py`. Not very important though